check-plugins/check_gitlab_runner_jobs/check_gitlab_runner_jobs.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+import httpx
+import argparse
+import sys
+
+"""
+Copyright 2023 Deutsche Telekom MMS GmbH
+Maintainer: Julian Mühmelt
+This script is an icinga2 monitoring check to check how many jobs are in a given state per runner
+"""
+
+def runner_string_to_dict(string):
+    runner = {}
+    if string.find(",") != -1:
+        string_split = string.split(",")
+        for a in string_split:
+            name_id = a.split(":")
+            runner.update({name_id[0].strip(): name_id[1].strip()})
+    else:
+        string_split = string.split(":")
+        runner.update({string_split[0]: string_split[1]})
+    return runner
+
+# script
+if __name__ == "__main__":
+    # argparse configuration
+    parser = argparse.ArgumentParser(description="This script is an icinga2 monitoring check to check how many jobs are in a given state per runner")
+    parser.add_argument("-u", "--url", help="gitlab base url", required=True)
+    parser.add_argument("-s", "--jobstate", help="job state (which jobs should be counted) default=running", default="running")
+    parser.add_argument("-t", "--token", help="gitlab access token", required=True)
+    parser.add_argument("-r", "--runner", help="runner name:id (multiple separated by comma) format: name:id,name:id", required=True)
```

Comment:
When adding `formatter_class=argparse.ArgumentDefaultsHelpFormatter`, the defaults are always shown and you don't need to add them to the help output.
```suggestion
    parser = argparse.ArgumentParser(
        description="This script is an icinga2 monitoring check to check how many jobs are in a given state per runner",
        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
    )
    parser.add_argument("-u", "--url", help="gitlab base url", required=True)
    parser.add_argument("-s", "--jobstate", help="job state (which jobs should be counted)", default="running")
    parser.add_argument("-t", "--token", help="gitlab access token", required=True)
    parser.add_argument("-r", "--runner", help="runner name:id (multiple separated by comma) format: name:id,name:id", required=True)
```